experiments/mjc_mdgps_example/hyperparams.py

```diff
@@ -12,7 +12,7 @@
 from gps.algorithm.cost.cost_fk import CostFK
 from gps.algorithm.cost.cost_action import CostAction
 from gps.algorithm.cost.cost_sum import CostSum
-from gps.algorithm.cost.cost_utils import RAMP_FINAL_ONLY
+from gps.algorithm.cost.cost_utils import RAMP_FINAL_ONLY, evall1l2term
```

Comment:
We don't use this in the SSRL work, right? If so, then there shouldn't be any changes in this file.